src/region.js

```diff
@@ -84,9 +84,11 @@ const Region = MarionetteObject.extend({
     }
   },
 
-  _attachView(view) {
+  _attachView(view, options = {}) {
     const shouldTriggerAttach = !view._isAttached && isNodeAttached(this.el);
-    const shouldReplaceEl = !!this.getOption('replaceElement');
+    const shouldReplaceEl = _.defaults(options, {
```

Comment:
thoughts on `const shouldReplaceEl = _.isBoolean(options.shouldReplaceEl) ? options.shouldReplaceEl : !!this.getOption('replaceElement');`